src/pages/Home/Home.js

```diff
@@ -0,0 +1,90 @@
+import React, {useEffect, useState} from 'react';
+import axios from "axios";
+import {Link, NavLink} from "react-router-dom";
+import Redditlogo from "../../assets/logo.png"
+
+function Home(props) {
+const [endpoint,setEndpoint] = useState(`https://www.reddit.com/hot.json?limit=15`)
```

Comment:
De setter wordt nooit aangeroepen. Daarom kun je de link gewoon als constante opnemen
```suggestion
const endpoint = `https://www.reddit.com/hot.json?limit=15`;
```

En het liefst dan buiten het component zetten, omdat deze nooit zal veranderen, dus dan hoeft die ook niet binnen het component te staan